esp/services/espcontrol/ws_espcontrolservice.cpp

```diff
@@ -192,8 +193,27 @@ bool CWSESPControlEx::onSetLogging(IEspContext& context, IEspSetLoggingRequest&
         if (!m_container)
             throw MakeStringException(ECLWATCH_INTERNAL_ERROR, "Failed to access container.");
 
-        if (!req.getLoggingLevel_isNull())
-            m_container->setLogLevel(req.getLoggingLevel());
+        if (version >= 1.04)
+        {
+            unsigned requestedLevel = req.getLoggingLevelEnum();
+            unsigned espLevel;
+
+            switch (requestedLevel)
+            {
+                case 0:   espLevel = LogNone;    break;
```

Comment:
The standard for the platform code seems to add new lines for all cases in a switch